angular.json

```diff
@@ -93,6 +93,9 @@
           "configurations": {
             "production": {
               "buildTarget": "ng-zorro-antd-doc:build:production"
+            },
+            "development": {
+              "buildTarget": "ng-zorro-antd-doc:build:development"
```

Comment:
```json
"configurations": {
  "production": {
    "browserTarget": "ceres-web:build:production"
  },
  "development": {
    "browserTarget": "ceres-web:build:development"
  }
},
"defaultConfiguration": "development"
```

You also need to add a `defaultConfiguration`.

Reply:
> ```diff
> "configurations": {
>   "production": {
>     "browserTarget": "ng-zorro-antd-doc:build:production"
>   },
>   "development": {
>     "browserTarget": "ng-zorro-antd-doc:build:development"
>   }
> },
> +"defaultConfiguration": "development"
> ```
> 
> You also need to add a `defaultConfiguration`.

I see, I misunderstood earlier. I thought 'npm run start' required the 'production' configuration.
I have now added the 'defaultConfiguration' as suggested.